### ISSUE TITLE
Setting seed fails

### ISSUE THREAD

**Issue by MrManor** [brinchj/RndPhrase#16]
Installed in 3 browsers (2 google Chrome and 2 in frefox) without problems , but after I installed in a second FIrefox on another machine I cannot set the seed on the latest Firefox instance. Every time I try to enter a letter in the seed setting I get two pop ups:

Error: More than 1 seed stored in state (2)

I dont think the two Firefox instances are on any way synchronized.

**Comment by brinchj** [comment]
Hi,

I don't have much time available for debugging this and I personally use Chrome where the extension is working :-)

If you have the time, you can try debugging this yourself, or you may try this fork of the project:
https://github.com/RndPhrase

I am personally migrating to the password-store utility:
https://www.passwordstore.org/

While more intrusive than RndPhrase it seems simpler in the long-run (although it requires more investment upfront).

**Comment by MrManor** [comment]
Well, I decided to ditch my Firefox profile on the machine i question and cleared out .mozilla/firefox/. That also fixed this problem.

I have tried password-store and while it works, I know I will never be able to learn my non-computer friends or family to use anything like that. I certainly hope that RndPhrase will stay alive for a while - and my small donation to urge for for a mobile edition was no just a loss.
